pkg/test/utils/kubernetes.go

```diff
@@ -938,6 +938,17 @@ func ExpandEnv(c string, env map[string]string) string {
 func GetArgs(ctx context.Context, cmd harness.Command, namespace string, env map[string]string) (*exec.Cmd, error) {
 	argSlice := []string{}
 
+	if cmd.Command != "" && cmd.Script != "" {
+		return nil, errors.New("command and script can not be set in the same configuration")
+	}
+	if cmd.Command == "" && cmd.Script == "" {
+		return nil, errors.New("command or script must be configured")
+	}
+
```

Comment:
```suggestion
if cmd.Script != "" && cmd.Namespaced {
      return nil, errors.New("script can not used namespaced, use the NAMESPACE environment variable instead"
}
```

I think we should make sure that users don't try to combine these

Reply:
I'm beginning to think `namespaced` is a bad idea... we have better ways of working with namespace now... when we need it.   I'll add this in.  thanks!